src/view/MainViewManager.js

```diff
@@ -1683,8 +1683,11 @@ define(function (require, exports, module) {
 
         return result;
     }
-    
-    
+
+    function _providerAvailableForPos(editor){
```

Comment:
I don't think this belongs on the `MainViewManager`.  Can't you leave it all on the `EditorManager`?